src/components/AccountSwitcher.tsx

```diff
@@ -201,7 +201,7 @@ function AccountSwitcher() {
                         }
                         item.onSelected();
                     }}
-                    containerStyles={{maxHeight: windowHeight / 2, width: 'fit-content', maxWidth: '100%'}}
+                    containerStyles={[{maxHeight: windowHeight / 2, width: 'fit-content'}, styles.pb0, styles.mw100]}
```

Comment:
Create a style object for `width: 'fit-content'` in `src/styles/utils/sizing.ts` and use it